### PR TITLE
Added Kadane's Algorithm(DP) in Dynamic Programming sub-folder inside Algorithms folder.

### PR DESCRIPTION
**Pull request by vishal-bhola**

### CODE REVIEW

**Comment by vishal-bhola** [comment]
Please review it.

**Comment by vishal-bhola** [comment]
I have to make more PR's . So please review it.

**Comment by vishal-bhola** [comment]
@Bhavnaharitsa Please review my PR as i have to make more PR's.

**Comment by Bhavnaharitsa** [comment]
Add sample input output in comment section...name of the algo

**Comment by VersatileVishal** [comment]
Could you please elaborate.

File: Algorithms/DynamicProgramming(DP)/Kadane's Algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        global=current;
+    }
+    
+    return global;
+}
+
+int main() {
+	int t;
+	cin>>t;;
+	while(t--)
```
Comment (Bhavnaharitsa) [inline Algorithms/DynamicProgramming(DP)/Kadane's Algorithm.cpp]: please avoid using single letter variables and replace with sensible names

Comment (vishal-bhola) [reply Algorithms/DynamicProgramming(DP)/Kadane's Algorithm.cpp]: Changes done as per the requirements.

Comment (vishal-bhola) [reply Algorithms/DynamicProgramming(DP)/Kadane's Algorithm.cpp]: Please review.

**Pull request merged**